crossbuild/cmake_toolchains/x86_64-unknown-freebsd.toolchain

```diff
@@ -0,0 +1,19 @@
+# Toolchain file for x86_64-unknown-freebsd
+set(CMAKE_SYSTEM_NAME FreeBSD)
+
+set(CMAKE_SYSROOT /opt/x86_64-unknown-freebsd/x86_64-unknown-freebsd/sys-root)
+set(CMAKE_INSTALL_PREFIX /)
+
+set(CMAKE_C_COMPILER /opt/x86_64-unknown-freebsd/bin/clang)
+set(CMAKE_CXX_COMPILER /opt/x86_64-unknown-freebsd/bin/clang++)
+set(CMAKE_LINKER /opt/super_binutils/bin/ld)
+set(SHARED_FLAGS "--target=x86_64-unknown-freebsd11.1 -fuse-ld=${CMAKE_LINKER} --sysroot=${CMAKE_SYSROOT} -I${CMAKE_SYSROOT}/include/")
```

Comment:
Why is it necessary to set these flags?  I should think that all of these options are baked into the `clang` we build.